vividus-plugin-web-app/src/main/java/org/vividus/ui/web/listener/WebSourceCodePublishingOnFailureListener.java

```diff
@@ -99,7 +99,7 @@ private String getElementSource(SearchContext searchContext)
     {
         try
         {
-            return ((WebElement) searchContext).getAttribute("innerHTML");
+            return webJavascriptActions.executeScript("return arguments[0].outerHTML;", searchContext);
         }
         catch (StaleElementReferenceException exception)
```

Comment:
will it be still stale element reference exception?

Reply:
yes, it will